source/_posts/announcing-eventy.md

```diff
@@ -4,33 +4,33 @@ section: content
 title: Announcing Eventy
 date: 2019-10-19
 ---
-Last Tuesday I finally announced my next passion project, [Eventy](https://eventy.io). I must say that the timing was a little off. I've been planning on doing the announcement for several weeks now but every time something kept creeping in between. So when I saw the announcement by Meetup.com's new pricing plans and all of the outrage by that, I knew that there wouldn't be a better moment then that.
+Last Tuesday, I finally announced my next passion project, [Eventy](https://eventy.io). I must say that the timing was a little off. I've been planning on doing the announcement for several weeks now, but every time something kept creeping in between. So when I saw the announcement by Meetup.com's new pricing plans and all of the outrages by that, I knew that there wouldn't be a better moment than that.
 
-Eventy is an app that I've been thinking of building for quite a while now. But over the past years the idea has been becoming more and more concrete. The thing that I sorely miss, being a user group leader, conference organiser, speaker and attendee is a unified platform. A place where I can maintain and keep track of all these different parts of an event experience. All while having a really nice end user experience.
+Eventy is an app that I've been thinking of building for quite a while now. But over the past years, the idea has been becoming more and more concrete. The thing that I sorely miss, being a user group leader, conference organizer, speaker, and attendee, is a unified platform. A place where I can maintain and keep track of all these different parts of an event experience. All while having a really nice end-user experience.
```

Comment:
Organiser is the British spelling while Organizer is the US one so it's actually still correct. But I don't mind editing it here